src/game/Player.h

```diff
@@ -405,6 +405,13 @@ struct ARXCHARACTER {
 	
 };
 
+/**
+ * player.skin goes from 0 to NUMBER_OF_PLAYER_SKINS - 1
+ */
+constexpr short NUMBER_OF_PLAYER_SKINS = 10;
+constexpr short MX_PLAYER_SKIN = NUMBER_OF_PLAYER_SKINS;
```

Comment:
nope, because skins go from 0 to 9 and so the number of player skins is 10
but then again the MAX cheat player skin is the next after 9 so it's also 10

Reply:
MX_PLAYER_SKIN refers to the MAX CHEAT skin (like the cur_mx variable), it's not referring to the maximum player skin, so that's a valid point there that can cause confusion. I'll correct it in a second.

Reply:
renamed it to `MAX_CHEAT_PLAYER_SKIN`